src/apps/frontend/components/button/index.tsx

```diff
@@ -1,26 +1,28 @@
 import clsx from 'clsx';
 import React, { PropsWithChildren } from 'react';
 
-import { ButtonKind, ButtonType } from '../../types/button';
+import { ButtonKind, ButtonSize, ButtonType } from '../../types/button';
 import Spinner from '../spinner/spinner';
 
 import styles from './button.styles';
 
 interface ButtonProps {
   disabled?: boolean;
   isLoading?: boolean;
+  kind?: ButtonKind;
   onClick?: (e) => void;
+  size?: ButtonSize;
   type?: ButtonType;
-  kind?: ButtonKind;
 }
 
 const Button: React.FC<PropsWithChildren<ButtonProps>> = ({
   children,
   disabled,
   isLoading,
+  kind = ButtonKind.PRIMARY,
   onClick,
+  size,
```

Comment:
Can you assign a default value to `size` just like kind and type?

Reply:
According to our team, the button component they created the secondary button without padding, if I apply the default size to the button, then it also reflects on the login page which is not properly align  as shown in the below image.

![image](https://github.com/jalantechnologies/boilerplate-mern/assets/126680323/01ba7c53-03e1-4bac-8f43-c5aa51078549)